### PR TITLE
Update the archetypes documentation

### PR DESCRIPTION
**Pull request by olafleur**
- Switch to archetype version 1.5-SNAPSHOT
- Remove links to deleted archetypes
- Update GWTP's URL documentation

### CODE REVIEW

File: archetypes/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Create a GWT-Platform project from a project template listed below.
 ##GWTP Reference
 * [GWTP Custom Google Search](http://www.google.com/cse/home?cx=011138278718949652927:5yuja8xc600) - Search GWTP documentation, wiki and thread collections.
```
Comment (Chris-V) [inline archetypes/README.md]: This should be removed as well

**Comment by Chris-V** [comment]
Lgtm

File: archetypes/README.md (hunk 1 above)
Comment (olafleur) [inline archetypes/README.md]: Why? From what I can see it still seems to work.

Comment (Chris-V) [inline archetypes/README.md]: results are limited to github

Comment (olafleur) [inline archetypes/README.md]: Ok, removing, then. ;-)

**Pull request merged**